webextensions/api/tabs.json

```diff
@@ -2758,6 +2758,25 @@
                 }
               }
             },
+            "autoDiscardable": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "54"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "116"
+                  },
+                  "firefox_android": "mirror",
```

Comment:
This property isn't really supported on Firefox for Android. Similar to some other properties which are marked as unsupported in BCD, we return a dummy value to improve extension compat, but the property isn't really supported, so this should be marked as unsupported in Firefox for Android.